README.md

```diff
@@ -104,6 +104,18 @@ The following tools must be installed:
 - git: [installation instructions](https://git-scm.com/downloads/linux)
 - make: install by running `apt-get -y install make` (`sudo` might be required)
 
+### Additional prerequisites to enable bash completion
+
+- Install [argcomplete](https://pypi.org/project/argcomplete/) globally on your machine.
+  ```shell
+  pip install argcomplete
+  activate-global-python-argcomplete
```

Comment:
Do I have to do it once, or every time I create a new terminal? What about the other steps?